mlrun/runtimes/funcdoc.py

```diff
@@ -206,6 +206,10 @@ def ann_type(ann):
 
 
 def iter_elems(ann):
+    if hasattr(ann, "elts"):
+        return ann.elts
+    if hasattr(ann, "id"):
+        return [ann.id]
```

Comment:
would be nice to add a comment here if you already touched it :)

Reply:
good thing you said this - I found a bug and fixed it :)